src/app/modals/BasicAboutModal.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```

Comment:
Please, for consistency use double quotes "

Reply:
This is a style choice we should discuss as a team and enforce with pre-merge linting (I'm fine with either).

Reply:
I've intended to replace all unneeded double quotes to single in further PRs, so maybe we want to keep the single quote convention?

Reply:
They are functionally the same, and single quotes are more common. I've always preferred them when writing JS, so that would take my vote. I would suggest ignoring the issue for this PR, and following up with another PR that converts the project to whatever it is that we decide, and also introducing linting that enforces the rule and is integrated with CI.